packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -60,7 +60,7 @@ const defaultRouterliciousDriverPolicies: IRouterliciousDriverPolicies = {
 export class RouterliciousDocumentServiceFactory implements IDocumentServiceFactory {
 	private readonly driverPolicies: IRouterliciousDriverPolicies;
 	private readonly blobCache: ICache<ArrayBufferLike>;
-	private readonly wholeSnapshotTreeCache: ICache<INormalizedWholeSummary> = new NullCache();
+	private readonly wholeSnapshotTreeCache: ICache<INormalizedWholeSnapshot> = new NullCache();
 	private readonly shreddedSummaryTreeCache: ICache<ISnapshotTreeVersion> = new NullCache();
```

Comment:
>shreddedSummaryTreeCache [](http://example.com/codeflow?start=18&length=24)

should this be snapshot as well ?

Reply:
The concept of shredded summary is both on download and upload. So one way is to just rename it to shreddedTreeCache but this shredded is not used anymore. So I left it as is for now.